### DISCUSSION TITLE
Landing page change

### DISCUSSION THREAD

**Discussion by RobLW** [immich-app/immich#12004] [General]
When I first open "photos" menu, it is showing everything in date order (most recent at top) which is perfect in terms of ordering. But if I only wanted to see photos, I currently have to use the filter in the search bar and select images only. But now I don't have a time line and not sure what the order of the images is. 

Whether on the landing page, or within the search filter, would it please be possible to allow the user to specify the ordering preference from a drop down menu which might use various meta-data such as "date create, camera, rating," etc. This sort of thing is foundational and more important to get in place well before ML stuff.

**Comment by bo0tzz** [comment]
#10400
